package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "client": "yarn workspace client start",
     "server": "yarn workspace server start",
-    "start": "concurrently --kill-others-on-fail \"yarn server\"  \"yarn client\""
+    "shared": "yarn workspace server start",
```

Comment:
Why is this the same as `yarn server`? Does `shared` need to be part of any scripts anyway?